winappdbg/system.py

```diff
@@ -530,7 +530,7 @@ def simple_debugger( argv ):
             # There may be more than one, so we'll sort out later which one to load.
             candidates = []
 
-            if win32.arch in (win32.ARCH_I386, win32.ARCH_AMD64):
+            if win32.arch in (win32.ARCH_I386, win32.ARCH_AMD64, win32.ARCH_ARM64):

```

Comment:
Does this old method of getting dbghelp.dll actually work on ARM? I thought ARM support for Windows was added much later...